config/v1/types_console.go

```diff
@@ -7,24 +7,27 @@ import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
 // Console holds cluster-wide information about Console.  The canonical name is `cluster`
-// TODO this object is an example of a possible grouping and is subject to change or removal
 type Console struct {
 	metav1.TypeMeta `json:",inline"`
 	// Standard object's metadata.
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
 	// spec holds user settable values for configuration
-	Spec ConsoleSpec `json:"spec"`
+	// +optional
```

Comment:
Spec should not be optional, internals in spec can be, but spec should not.

Reply:
Great will update